runhouse/main.py

```diff
@@ -64,6 +64,15 @@
     help=f"Cluster related CLI commands. For more info run {ITALIC_BOLD}runhouse cluster --help{RESET_FORMAT}"
 )
 
+app.add_typer(cluster_app, name="cluster")
+
+# creating a cluster app, so we could create subcommands of cluster (i.e runhouse cluster list).
+# Register it with the main runhouse application
+server_app = typer.Typer(
+    help=f"Runhouse server related CLI commands. For more info run {ITALIC_BOLD}runhouse server --help{RESET_FORMAT}"
```

Comment:
The --help message seems duplicative since the "--help" shows up automatically, unless there's another reason to include it? (same for `cluster` too btw)

```suggestion
    help="Runhouse server related CLI commands."
```